okapi-core/src/main/java/org/folio/okapi/managers/PullManager.java

```diff
@@ -52,17 +52,17 @@ private void getRemoteUrl(Iterator<String> it,
     final Buffer body = Buffer.buffer();
     HttpClientRequest req = httpClient.getAbs(url, res1 -> {
       if (res1.failed()) {
-        logger.info("pull for " + baseUrl + " failed with status "
-          + res1.cause().getMessage());
+        logger.warn("pull for {} failed with status {}", baseUrl,
+          res1.cause().getMessage());
```

Comment:
```suggestion
        logger.warn("pull for {} failed: {}", baseUrl,
          res1.cause().getMessage(), res1);
```
Add stacktrace.

Reply:
There's no stacktrace (not an exception)